tests/search/tensor_feed/tensor_feed.rb

```diff
@@ -21,6 +21,18 @@ def test_tensor_json_feed
     puts "search_docs: #{search_docs}"
     assert_tensor_docs(search_docs)
 
+    visit_response = vespa.document_api_v1.visit(:selection => "test.my_tensor", :fieldSet => "test:[document]", :cluster => "search", :wantedDocumentCount => 10)
```

Comment:
Consider adding a helper function that takes the visit parameters, visits, prints, and then returns the visit docs to reduce code duplication.

Reply:
Done